sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_exporter.py

```diff
@@ -45,6 +45,10 @@
     "code.",
 ]
 
+_STANDARD_AZURE_MONITOR_ATTRIBUTES = [
```

Comment:
I was thinking to maybe add something at the beginning of the name to ensure those are not added in the envelope properties, we may need different ones to sampleRate and with the name you don't need to mantain the list here. Something like "azureMonitor.sampleRate" or similar, just a suggestion.

Reply:
Using "_MS.sampleRate"